curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-140-modified-fibonacci-golden-nuggets.md

```diff
@@ -8,35 +8,41 @@ dashedName: problem-140-modified-fibonacci-golden-nuggets
 
 # --description--
 
-Consider the infinite polynomial series AG(x) = xG1 + x2G2 + x3G3 + ..., where Gk is the kth term of the second order recurrence relation Gk = Gk−1 + Gk−2, G1 = 1 and G2 = 4; that is, 1, 4, 5, 9, 14, 23, ... .
+Consider the infinite polynomial series $A_G(x) = xG_1 + x^2G_2 + x^3G_3 + \cdots$, where $G_k$ is the $k$th term of the second order recurrence relation $G_k = G_{k − 1} + G_{k − 2}, G_1 = 1$ and $G_2 = 4$; that is, $1, 4, 5, 9, 14, 23, \ldots$.
 
-For this problem we shall be concerned with values of x for which AG(x) is a positive integer.
+For this problem we shall be concerned with values of $x$ for which $A_G(x)$ is a positive integer.
 
 The corresponding values of x for the first five natural numbers are shown below.
 
-xAG(x) (√5−1)/41 2/52 (√22−2)/63 (√137−5)/144 1/25
+| $x$                         | $A_G(x)$ |
+|-----------------------------|----------|
+| $\frac{\sqrt{5} − 1}{4}$    | $1$      |
+| $\frac{2}{5}$               | $2$      |
+| $\frac{\sqrt{22} − 2}{6}$   | $3$      |
+| $\frac{\sqrt{137} − 5}{14}$ | $4$      |
+| $\frac{1}{2}$               | $5$      |
 
-We shall call AG(x) a golden nugget if x is rational, because they become increasingly rarer; for example, the 20th golden nugget is 211345365. Find the sum of the first thirty golden nuggets.
+We shall call $A_G(x)$ a golden nugget if $x$ is rational, because they become increasingly rarer; for example, the 20th golden nugget is 211345365. Find the sum of the first thirty golden nuggets.
```

Comment:
```suggestion
We shall call $A_G(x)$ a golden nugget if $x$ is rational because they become increasingly rarer; for example, the 20th golden nugget is 211345365. Find the sum of the first thirty golden nuggets.
```